src/mattack_actors.cpp

```diff
@@ -304,7 +304,7 @@ bool gun_actor::call( monster &z ) const
     if( z.friendly ) {
         int max_range = 0;
         for( const auto &e : ranges ) {
-            max_range = std::max( { max_range, e.first.first, e.first.second } );
+            max_range = std::max( std::max( max_range, e.first.first ), e.first.second );
```

Comment:
It requires an extra header. It was cleaner to break it up than to include a header just for 3 way max.